plugins/inputs/cpufreq/cpufreq.go

```diff
@@ -0,0 +1,120 @@
+package cpufreq
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const defaultHostSys = "/sys"
+
+type CPUFreq struct {
+	PathSysfs       string `toml:"host_sys"`
+	GatherThrottles bool   `toml:"gather_throttles"`
+}
+
+var sampleConfig = `
+  ## Path for sysfs filesystem.
+  ## See https://www.kernel.org/doc/Documentation/filesystems/sysfs.txt
+  ## Defaults:
+  # host_sys = "/sys"
+  ## Gather CPU throttles per core
```

Comment:
Add a newline here for readability
```suggestion
  # host_sys = "/sys"
  ##
  ## Gather CPU throttles per core
```